controllers/update_pod_config.go

```diff
@@ -102,14 +105,16 @@ func (updatePodConfig) reconcile(r *FoundationDBClusterReconciler, context ctx.C
 		if !synced {
 			allSynced = false
 			hasUpdate = true
-			curLogger.Error(err, "Update Pod ConfigMap annotation")
+			if err != nil {
+				curLogger.Error(err, "Update Pod ConfigMap annotation")
+			}
 			if internal.IsNetworkError(err) {
 				processGroup.UpdateCondition(fdbtypes.SidecarUnreachable, true, cluster.Status.ProcessGroups, processGroup.ProcessGroupID)
 			} else {
 				processGroup.UpdateCondition(fdbtypes.IncorrectConfigMap, true, cluster.Status.ProcessGroups, processGroup.ProcessGroupID)
 			}
 
-			pod.ObjectMeta.Annotations[fdbtypes.OutdatedConfigMapKey] = time.Now().Format(time.RFC3339)
+			pod.ObjectMeta.Annotations[fdbtypes.OutdatedConfigMapKey] = fmt.Sprintf("%d", time.Now().Unix())
```

Comment:
What is the benefit of this change? I would prefer the previous format to have it in aa human-readable format.

Reply:
The kubernetes monitor is reading this annotation and parsing its contents, and it is currently expecting an integer timestamp. If you think it's worthwhile, we can modify that to expect a different format.